packages/admin/admin/src/index.ts

```diff
@@ -201,18 +201,18 @@ export { useStoredState } from "./hooks/useStoredState";
 export { InputWithPopper, InputWithPopperComponents, InputWithPopperProps } from "./inputWithPopper/InputWithPopper";
 export { InputWithPopperClassKey } from "./inputWithPopper/InputWithPopper.slots";
 export { messages } from "./messages";
+export { MainNavigationCollapsibleItem, MainNavigationCollapsibleItemProps } from "./mui/mainNavigation/CollapsibleItem";
+export { MainNavigationCollapsibleItemClassKey } from "./mui/mainNavigation/CollapsibleItem.styles";
+export { MainNavigationContext, MainNavigationContextValue, WithMainNavigation, withMainNavigation } from "./mui/mainNavigation/Context";
+export { MainNavigationItem, MainNavigationItemProps } from "./mui/mainNavigation/Item";
+export { MainNavigationItemClassKey } from "./mui/mainNavigation/Item.styles";
+export { MainNavigationItemAnchorLink, MainNavigationItemAnchorLinkProps } from "./mui/mainNavigation/ItemAnchorLink";
+export { MainNavigationItemGroup, MainNavigationItemGroupClassKey, MainNavigationItemGroupProps } from "./mui/mainNavigation/ItemGroup";
+export { MainNavigationItemRouterLink, MainNavigationItemRouterLinkProps } from "./mui/mainNavigation/ItemRouterLink";
+export { MainNavigation, MainNavigationProps } from "./mui/mainNavigation/MainNavigation";
+export { MainNavigationClassKey } from "./mui/mainNavigation/MainNavigation.styles";
```

Comment:
Now maybe a good time to ask whether we actually need all these exports. For instance, none of our projects use withMenu. @jamesricky what do you think?

Reply:
We can remove them in a later PR.